shell/platform/android/io/flutter/embedding/engine/systemchannels/LocalizationChannel.java

```diff
@@ -48,4 +97,19 @@ public void sendLocales(@NonNull List<Locale> locales) {
     }
     channel.invokeMethod("setLocale", data);
   }
+
+  /**
+   * Handler that receives platform messages sent from Flutter to Android through a given {@link
+   * PlatformChannel}.
+   *
+   * <p>To register a {@code LocalizationMessageHandler} with a {@link PlatformChannel}, see {@link
+   * LocalizationChannel#setLocalizationMessageHandler(LocalizationMessageHandler)}.
+   */
+  public interface LocalizationMessageHandler {
+    /**
+     * The Flutter application would like to obtain the string resource of given {@code key} in
```

Comment:
Rephrase to be less passive/third person.

perhaps `Obtains the string resource of the provided {@code key} ...`

Reply:
I copied the tone from `PlatformChannel.java`.

The way I understand it is this comment is meant for the implementor of this interface.

What do you think?